### DISCUSSION TITLE
Fetching grammars

### DISCUSSION THREAD

**Discussion by sascha-andres** [helix-editor/helix#5583] [General]
I try to fetch grammars, but all grammars fail to fetch with cannot create directory like this:

     Failure 123/123: Could not create grammar directory "/tmp/.mount_hxkLAvdb/usr/lib/helix/runtime/grammars/sources/bicep"

Besides that /tmp has no special perms, can I override the temp dir helix is using fot his?

**Comment by archseer** [comment]
Probably a version of https://github.com/helix-editor/helix/issues/3346 since the files seem to be under `/usr/lib`

**Comment by sascha-andres** [comment]
I tried running hx as sudo, but this led to the same error

**Comment by archseer** [comment]
You shouldn't run as sudo as this will mess up your package installation. Are you trying to add new grammars or why are you trying to fetch? Right now you need to install from source and set a custom runtime path.

This will get easier once we support multiple runtime paths: https://github.com/helix-editor/helix/pull/5411
